win32/glob.c

```diff
@@ -170,19 +206,27 @@ PHPAPI int glob(const char *pattern, int flags, int (*errfunc)(const char *, int
 	flags |= GLOB_NOESCAPE;
 #endif
 
-	patnext = (uint8_t *) pattern;
+	patnext = (u_char *) pattern;
```

Comment:
`uint8_t` is a C99 standard type.

Reply:
It's this way in the OpenBSD source; must have been changed in PHP's fork of it. It should match the older PHP version though for portability.